src/routes/hub-interfaces.tsx

```diff
@@ -0,0 +1,53 @@
+import { Breadcrumb } from "antd";
+import React, { useState } from "react";
+import { Link, useNavigate, useParams } from "react-router-dom";
+import InterfacesContainer, {
+  InterfacesView,
+} from "../hub/Interfaces.container";
+import Page from "../layout/Page";
+import { AppstoreOutlined, BarsOutlined } from "@ant-design/icons";
+
+const activeStyle = { fontSize: 22 };
+const inactiveStyle = { ...activeStyle, color: "rgba(0, 0, 0, 0.2)" };
+
+function NewInterfacesCatalog() {
+  const [view, setView] = useState<InterfacesView>(InterfacesView.List);
```

Comment:
As I suggested in other comment, it would be good to preserve selected view. However, maybe let's skip it for now, as this should be saved in local storage - let's do it later 👍